### PR TITLE
Add support to get dice results

### LINKED ISSUE #164: Support for DICE message type.
Currently there is new message type called [Dice](https://core.telegram.org/bots/api#dice). Would be nice to have that.

### PR DESCRIPTION
**Pull request by daron666**
Closes #164

### CODE REVIEW

**Comment by daron666** [comment]
@augustjune Please review and release :)

File: examples/src/main/scala/samples/DiceExample.scala [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        Bot.polling[IO].follow(echos)
+      }
+      .compile
+      .drain
+      .as(ExitCode.Success)
+
+  def echos[F[_]: Sync: TelegramClient]: Scenario[F, Unit] =
+    for {
+      msg <- Scenario.expect(any)
+      _   <- Scenario.eval(SendDice(msg.chat.id, Dice).call[F, DiceThrownMessage])
```
Comment (augustjune) [inline examples/src/main/scala/samples/DiceExample.scala:30]: I think it's worth adding another case of `MessageContent` with `DiceEmoji` to provide convenient syntax for sending dice/darts: `Scenario.eval(chat.send(Dice))`

**Comment by augustjune** [comment]
Great PR @daron666, thanks for your contribution 🙏

File: examples/src/main/scala/samples/DiceExample.scala (hunk 1 above)
Comment (daron666) [reply examples/src/main/scala/samples/DiceExample.scala:30]: Ok, gonna add.

Comment (daron666) [reply examples/src/main/scala/samples/DiceExample.scala]: @augustjune done

**Comment by daron666** [comment]
Kind a exposed my token. Nevermind, already revoked it :) lol

**Pull request merged**